frontend/src/features/properties/map/acquisition/add/AddAcquisitionContainer.tsx

```diff
@@ -21,6 +24,32 @@ export const AddAcquisitionContainer: React.FC<IAddAcquisitionContainerProps> =
   const formikRef = useRef<FormikProps<AcquisitionForm>>(null);
 
   const close = useCallback(() => onClose && onClose(), [onClose]);
+  const { selectedResearchFeature, setSelectedResearchFeature } = React.useContext(
+    SelectedPropertyContext,
+  );
+  console.log(selectedResearchFeature);
```

Comment:
loggerino

Reply:
This might also be copy pasta from research file as this form is all about Acquisition File